bluesky/plan_stubs.py

```diff
@@ -689,44 +689,58 @@ def configure(obj, *args, **kwargs):
     return (yield Msg('configure', obj, *args, **kwargs))
 
 
-def stage(obj):
+def stage(obj, *, group=None, wait=False):
     """
     'Stage' a device (i.e., prepare it for use, 'arm' it).
 
     Parameters
     ----------
     obj : Device
+    group : string (or any hashable object), optional
+        identifier used by 'wait'; None by default
+    wait : boolean, optional
+        If True, wait for completion before processing any more messages.
+        False by default.
 
     Yields
     ------
     msg : Msg
-        Msg('stage', obj)
 
     See Also
     --------
     :func:`bluesky.plan_stubs.unstage`
     """
-    return (yield Msg('stage', obj))
+    ret = yield Msg('stage', obj, group=group)
+    if wait:
```

Comment:
@danielballan should we only do a wait if `isinstance(ret, Status)`? Or is it harmless?